webapp/lib/view.dart

```diff
@@ -472,6 +480,7 @@ class ContentView {
         _conversationTabLink.classes.add('active');
       break;
     }
+    if (currentTabView != null ) projectSelectorView.populateProjects(controller.projectList);
```

Comment:
nit: extraneous space between `null` and `)`

Reply:
When is `currentTabView` null? how come this hasn't been an issue so far?

Reply:
Fixed the spacing.

So I've removed this check for null. There were instances where currentTabView was appearing null but I guess it was the UI which hadn't fully loaded. I've tested it again and looks fine now.